sympy/functions/elementary/complexes.py

```diff
@@ -580,6 +580,21 @@ def eval(cls, arg):
                 a, b = log(base).as_real_imag()
                 z = a + I*b
                 return exp(re(exponent*z))
+        if isinstance(arg, Expr):
+        # Handle other cases here as needed.
+          if arg.is_Add:
+              # If the argument is an addition expression, check each term.
+              new_args = []
+              for term in arg.args:
+                  if isinstance(term, Mul):
+                      # If the term is a multiplication expression, simplify it.
+                      simplified_term = term.simplify()
```

Comment:
Simplify should not be used in library code and especially not in an eval method.

Reply:
how should I solve it then??

Reply:
The issue is to do with evalf so something in evalf should be changed.